services/api-server/src/simcore_service_api_server/api/dependencies/authentication.py

```diff
@@ -14,8 +15,8 @@
 
 class Identity(BaseModel):
     user_id: PositiveInt
-    product_name: str
-    email: str
+    product_name: ProductName
+    email: EmailStr
```

Comment:
MINOR: use `LowerCaseEmailStr` which has a pre-processor to lowercase. This is the one we use everywhere to id users

Reply:
Fixed